src/chigame/users/models.py

```diff
@@ -14,10 +14,10 @@ class User(AbstractUser):
     """
 
     # First and last name do not cover name patterns around the globe
-    name = CharField(_("Name of User"), blank=True, max_length=255)
+    name = models.CharField(_("Name of User"), blank=True, max_length=255)
     first_name = None  # type: ignore
     last_name = None  # type: ignore
```

Comment:
if we are using a generic `name` attribute to be more accomodating of different global naming practices, what point is there in still having the first and last name as attributes?

Also what does having a `field = None` actually do?

Reply:
Django's built-in User model already includes a first_name and last_name attribute. The above code was generated by cookiecutter, but I believe it ensures that those two fields are going to always be set to None.